api/src/main/java/com/github/eyefloaters/console/api/model/ReplicaLocalStorage.java

```diff
@@ -0,0 +1,25 @@
+package com.github.eyefloaters.console.api.model;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+public record ReplicaLocalStorage(
+        @Schema(readOnly = true, description = "The total size of the log segments local to the replica, in bytes.")
+        long size,
+
+        @Schema(readOnly = true, description = """
+                The lag of the log's LEO with respect to the partition's high watermark
+                (if it is the current log for the partition) or the current replica's LEO
+                (if it is the `future` log for the partition).
+                """)
+        long offsetLag,
+
+        @Schema(readOnly = true, description = """
+                Whether this replica has been created by a AlterReplicaLogDirsRequest
+                but not yet replaced the current replica on the broker
```

Comment:
Do you think this is a better description?

```suggestion
                false this replica is the active log being used by the broker
                true if this replica represents a log being moved between log dirs on the broker  
```

Similarly I'm not totally sure `future` is the best name, even though it's the term used in the Admin API.

Reply:
I think that description is clearer. As far as the name, let's keep it as-is for now and revisit if necessary. As you said, it's the name from the Admin API, which at least helps keep things connected.